src/test/java/integration/gui/mock/ContainerAction.java

```diff
@@ -153,7 +158,7 @@ public void actionPerformed(ActionEvent e) {
                 }else if(e.getSource() == DemoGUITest.getInstance().getMetal()){
                     DemoGUITest.getInstance().changeThemeWith(new NimbusLookAndFeel());
                 }else if(e.getSource() == DemoGUITest.getInstance().getGtk()){
-                    DemoGUITest.getInstance().changeThemeWith(new GTKLookAndFeel());
+                    DemoGUITest.getInstance().changeThemeWith("com.sun.java.swing.plaf.gtk.GTKLookAndFeel");
```

Comment:
I think the better choice is used this code 

`UIManager.setLookAndFeel(UIManager.getCrossPlatformLookAndFeelClassName());`

Reply:
OK looks good - I will update but can't test this (no Linux here)

Reply:
I used linux, I can test it, thanks so much